src/uu/whoami/whoami.md

```diff
@@ -0,0 +1,7 @@
+# whoami
+
+```
+whoami
```

Comment:
I think we shouldn't ignore this in the code for consistency, even though the default output of clap is equivalent at the moment.

Reply:
(If we're consistent we can also make more changes in the future like providing a function in uucore to set up a command with correct usage, about, util name etc.)

Reply:
Oh actually it's not equivalent:
```
Usage: whoami
vs
Usage: target/debug/coreutils whoami
```